### PR TITLE
parsing for timestamp in load_current_steam_user()

### PR DESCRIPTION
**Pull request by cgcha**
Not ready for a review yet, but I'll put a draft out that "works".

It works correctly in getting the most recent user even with weird ordering in loginusers.vdf.

Case where this doesn't work: my friend logged in and definitely didn't have the correct launch configs, but the backend gamefinder tracer said they were.

### CODE REVIEW

**Comment by cgcha** [comment]
Issue noted in #39 whoops

**Comment by cgcha** [comment]
Setting this as ready to review because the noted bug is not related to this pr.

File: src/settings.rs [hunk 1]
```diff
@@ -112,24 +112,56 @@ impl Settings {
...
+                        .filter_map(|value| value.get_obj())
+                        .for_each(|user_data_obj| {
+                            user_data_obj.get("Timestamp")
+                                .and_then(|timestamp_values| timestamp_values.get(0))
+                                .and_then(|timestamp_vdf| timestamp_vdf.get_str())
+                                .and_then(|timestamp_str| timestamp_str.parse::<i64>().ok())
+                                .map(|timestamp| {
+                                    if timestamp > latest_timestamp {
+                                        latest_timestamp = timestamp;
+                                        latest_user_sid64 = Some(user_sid64.to_string());
```
Comment (Bash-09) [inline src/settings.rs:135]: Is there any reason to convert this to an int and then convert it back to a string only for it to be parsed back as an int later? May as well keep them as an int if possible.

**Review by Bash-09 — APPROVED** [review]

**Comment by Bash-09** [comment]
To fix [this issue](https://github.com/MegaAntiCheat/client-backend/issues/45) I might refactor some code surrounding this functionality and may touch this code a bit, but to start with I'm happy to merge this in.

**Pull request merged**